app/src/main/java/org/jellyfin/androidtv/details/DetailsOverviewPresenter.kt

```diff
@@ -52,17 +68,41 @@ class DetailsDescriptionPresenter : Presenter() {
 		val textStreamValue: TextView = view.details_description_streams_text_value
 	}
 
-	override fun onCreateViewHolder(parent: ViewGroup): ViewHolder {
+	override fun createRowViewHolder(parent: ViewGroup): ViewHolder {
 		val view = LayoutInflater
 			.from(parent.context)
 			.inflate(R.layout.row_details_description, parent, false)
 
 		return ViewHolder(view)
 	}
 
-	override fun onBindViewHolder(viewHolder: Presenter.ViewHolder, item: Any) {
+	override fun onBindRowViewHolder(viewHolder: RowPresenter.ViewHolder, row: Any) {
 		viewHolder as ViewHolder
-		item as BaseItem
+		row as DetailsOverviewRow
+		val item = row.item
+
+		// banner
+		//todo hide banner view when none found, support multiple banners
+		item.images.backdrops.firstOrNull()?.let {
+			// Android doesn't crop automatically but Glide does
+			// Picasso can also do this but doesn't read the XML attributes of the target view for it
+			// so the way Glide does it is preferred to avoid duplicate settings
+			Glide.with(viewHolder.view.context).load(it.url).into(viewHolder.banner)
```

Comment:
We probably should decide on one image handling library before we ship this

Reply:
I made a [separate issue](https://github.com/jellyfin/jellyfin-androidtv/issues/403) for this.